src/Assets.php

```diff
@@ -95,7 +95,7 @@ public static function register_assets() {
 		}
 
 		if ( Package::feature()->is_feature_plugin_build() ) {
-			$asset_api->register_script( 'wc-blocks-checkout', 'build/wc-blocks-checkout.js', [], false );
+			$asset_api->register_script( 'wc-blocks-checkout', 'build/blocks-checkout.js', [], false );
```

Comment:
wc-blocks-checkout/wc-checkout-block is going to be confusing. Can we be more specific in our naming here somehow?

Reply:
we discussed [naming here](https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/3613), we're trying to avoid conflict with core checkout script and our block checkout, if you have any idea of a good name we can implement it anytime because we still didn't publish this.